### PR TITLE
Remove deprecated methods

### PR DESCRIPTION
**Pull request by greysteil**
Looks like these were scheduled for deletion in 0.14.

### CODE REVIEW

File: spec/raven/integrations/rack_spec.rb [hunk 1]
```diff
@@ -102,7 +102,7 @@
     end
 
     stack = Raven::Rack.new(Rack::Lint.new(app))
-    expect { stack.call(env) }.to_not raise_error(Rack::Lint::LintError)
```
Comment (nateberkopec) [inline spec/raven/integrations/rack_spec.rb]: Can you remove this, just because it's unrelated?

**Comment by nateberkopec** [comment]
Can't even keep up with my own deprecation schedule!

**Comment by greysteil** [comment]
Updated!

**Comment by nateberkopec** [comment]
Kaboom.

**Pull request merged**